pkg/kubectl/cmd/util/helpers.go

```diff
@@ -427,6 +428,10 @@ func AddDryRunFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool("dry-run", false, "If true, only print the object that would be sent, without sending it.")
 }
 
+func AddIncludeUninitializedFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool(IncludeUninitializedFlag, false, "If true, include the object(s) that have not been realized yet. Such object(s) have a non-empty initializer list.")
```

Comment:
s/realized/initialized.
Change "Such object(s) have a non-empty initializer list." to "Only objects with empty metadata.initializers are regarded as initialized".